index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Elena Khalyavskaya | Junior Front-end Developer</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <div class="container-fluid container__header">
+        <div class="container" id="home">
+          <nav>
+            <ul>
+              <li><a href="#home">Home</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#skills">Skills</a></li>
+              <li><a href="#education">Education</a></li>
+              <li><a href="#my-works">My Works</a></li>
+            </ul>
+          </nav>
+          <div class="profile">
+            <img src="./img/my-pic.jpg" alt="My photo">
+            <h1>Elena Khalyavskaya</h1>
+            <h3>junior front-end developer</h3>
+            <ul>
+              <li><a href="mailto:lenach241@gmail.com"><i class="far fa-envelope"></i></a></li>
+              <li><a href="https://github.com/kasatka660" target="_blank"><i class="fab fa-github"></i></a></li>
+              <li><a href="https://www.facebook.com/lena.khalyavskaya" target="_blank"><i class="fab fa-facebook-f" ></i></a></li>
+              <li><a href="https://vk.com/lenakhalyavskaya" target="_blank"><i class="fab fa-vk"></i></a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="container about-me" id="about">
```

Comment:
Why did you register id here and below? I see that you did not use it for styling. This is not an error, just a little confusing to add attributes that are not used.

Reply:
All is well!
You can make marge.